examples/experimental/u7s.yaml

```diff
@@ -0,0 +1,145 @@
+# Deploy kubernetes via usernetes.
+# $ limactl start ./u7s.yaml
+# $ limactl shell u7s kubectl
+
+# It can be accessed from the host by exporting the kubeconfig file;
+# the ports are already forwarded automatically by lima:
+#
+# $ export KUBECONFIG=$(limactl list u7s --format 'unix://{{.Dir}}/copied-from-guest/kubeconfig.yaml')
+# $ kubectl get no
+# NAME           STATUS   ROLES           AGE   VERSION
+# u7s-lima-u7s   Ready    control-plane   33s   v1.28.0
+
+# This template requires Lima v0.8.0 or later
+images:
+# Try to use release-yyyyMMdd image if available. Note that release-yyyyMMdd will be removed after several months.
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release-20231010/ubuntu-22.04-server-cloudimg-amd64.img"
+  arch: "x86_64"
+  digest: "sha256:5bed3f233c2422187e86089deea51bb8469dc2a26e96814ca41ff8f14dc80308"
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release-20231010/ubuntu-22.04-server-cloudimg-arm64.img"
+  arch: "aarch64"
+  digest: "sha256:5167c1b13cb33274955e36332ecb7b14f02b71fd19a37a9c1a3a0f8a805ab8e5"
+# Fallback to the latest release image.
+# Hint: run `limactl prune` to invalidate the cache
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-amd64.img"
+  arch: "x86_64"
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-arm64.img"
+  arch: "aarch64"
+
+# Mounts are disabled in this template, but can be enabled optionally.
+mounts: []
+# containerd is managed by Docker, not by Lima, so the values are set to false here.
+containerd:
+  system: false
+  user: false
+provision:
+# See [KEP-2033](https://features.k8s.io/2033) Kubelet-in-UserNS (aka Rootless mode)
+- mode: system
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+    test -d /var/lib/usernetes && exit 0
+    branch=gen2-v20230919.0
+    curl -fsSL https://github.com/rootless-containers/usernetes/archive/$branch.tar.gz | tar -C /var/lib --transform="s/-$branch//" -xz
+    chown ${LIMA_CIDATA_UID} /var/lib/usernetes
+- mode: system
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+    command -v kubectl >/dev/null 2>&1 && exit 0
+    version=$(curl -L -s https://dl.k8s.io/release/stable.txt)
+    case $(uname -m) in
+      x86_64)   arch=amd64;;
+      aarch64)  arch=arm64;;
+    esac
+    curl -L "https://dl.k8s.io/release/$version/bin/linux/$arch/kubectl" -o /usr/local/bin/kubectl
+    chmod 755 /usr/local/bin/kubectl
+    kubectl version --client
+- mode: system
+  script: |
+    #!/bin/bash
+    cd /var/lib/usernetes/init-host
+    export CONTAINER_ENGINE=docker
+    ./init-host.root.sh
```

Comment:
Could you add a comment to explain that Docker is installed here?

Reply:
Could just remove it, and go with whatever default usernetes has